scripts/state.sh

```diff
@@ -8,7 +8,7 @@ read_option_from_state() {
     touch "$STATE_FILE"
   fi
 
-  local row=$(grep "^$1" "$STATE_FILE")
+  local row=$(cat "$STATE_FILE" | grep "$1")
```

Comment:
i don't think this changes anything, but the other one did not work with options for some reason